lib/mongoid/slug/criteria.rb

```diff
@@ -75,20 +75,23 @@ def objectid_slug_strategy id
       end
 
       # a string will always look like a slug
-      def string_slug_strategy id
+      def string_slug_strategy(_id)
         true
       end
 
-
       def for_slugs(slugs)
-        #_translations
-        localized = (@klass.fields['_slugs'].options[:localize] rescue false)
+        # _translations
+        localized = (begin
+                       @klass.fields['_slugs'].options[:localize]
+                     rescue
+                       false
+                     end)
```

Comment:
I think this is a great example why RuboCop is great. You just run `rubocop -a` and let Ruby style guide tell you what's more legible. This way we don't have to have to argue about it :)

Personally half the time I don't see that there's a `rescue` there and wonder how come `localized` is `false` when debugging.